charts/gardener/gardenlet/values.yaml

```diff
@@ -94,6 +94,9 @@ global:
         shootStateSync:
           concurrentSyncs: 5
           syncPeriod: 30s
+      resources:
+        capacity:
+          shoots: 200 # TODO: What is a good default value here?
```

Comment:
What about ~275? We have good experience with these values for now. /cc @vpnachev @vlerenc

Reply:
I would start with 250 for the default value or even less. But as this is configurable value, I have no strong opinion.

Reply:
Changed to 250